torch/csrc/jit/script/lexer.h

```diff
@@ -100,7 +100,9 @@ namespace script {
   _(TK_LIST_COMP, "list comprehension", "")      \
   _(TK_PASS, "pass", "pass")                     \
   _(TK_CLASS_DEF, "class", "class")              \
-  _(TK_IMPORT, "import", "import")
+  _(TK_IMPORT, "import", "import")               \
+  _(TK_NAMED_TUPLE, "NamedTuple", "NamedTuple")  \
```

Comment:
This should not be added as a token because it is not a python keyword.  It prevents someone from writing:

```
def foo():
   NamedTuple = 3
   return NamedTuple
```

which, while weird, is perfectly valid code.